### DISCUSSION TITLE
2023/03/22/%E4%B8%87%E6%B8%B8%E7%AC%94%E8%AF%95/

### DISCUSSION THREAD

**Discussion by giscus** [Zhuanjier/blog#12] [Announcements]
# 2023/03/22/%E4%B8%87%E6%B8%B8%E7%AC%94%E8%AF%95/

开发工程师笔试题  开始时间：2023-03-22 19:40        结束时间：2023-03-22 20:45  ¶题1 100 个小朋友围成一个圈，设定编号为 1~100，依次按 1、2、3、4、5、6、7、8、9 循环报数，报到 9 的出圈，直到所有小朋友出圈。请写代码打印出各个小朋友出圈顺序，语言不限。 ¶代码——C++ 123456789101112131415161718192

https://blog.zhuanjie.ltd/2023/03/22/%E4%B8%87%E6%B8%B8%E7%AC%94%E8%AF%95/

**Comment by RandallAnjie** [comment]
set 函数必须要满足这些算法：set属于关联容器，其元素相当于键值，使用 set 时，可以自动忽略重复的元素；而使用 vector 时可以保留重复的元素，vector属于顺序容器，其元素与存储位置与操作操作有关

顺序容器：list，queue，vector
关联容器：map，set
